velox/docs/configs.rst

```diff
@@ -521,6 +521,11 @@ Each query can override the config by setting corresponding query session proper
      - integer
      - 100
      - Maximum number of (bucketed) partitions per a single table writer instance.
+   * - hive.max-bucket-count
+     -
```

Comment:
Add the session property name here and above for hive.max-partitions-per-writers